OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -85,10 +87,29 @@ public void InvokeCommand(string name, string arg)
 					IssueDevCommand(world, "DevEnableTech");
 					IssueDevCommand(world, "DevFastCharge");
 					IssueDevCommand(world, "DevGiveCash");
+					IssueDevCommand(world, "DevLevelUp");
 					break;
 
 				case "crash":
 					throw new DevException();
+				
+				case "levelup":
+					var level = 0;
+					int.TryParse(arg, out level);
+
+					foreach (var actor in world.Selection.Actors)
+					{
+						var leveluporder = new Order("DevLevelUp", actor, false);
+						leveluporder.ExtraData = (uint)level;
+						
+						if (actor == null)
+							return;
```

Comment:
While that is definitely true we [do not remove null references](https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Selection.cs#L64), nor do we ensure only non-[null additions](https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Selection.cs#L22)